tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "esnext",
```

Comment:
This was changed to fix Typescript complaining about this syntax: `[...domNodeList]`

More details in slack: https://cloudfour.slack.com/archives/C03ECJ38E/p1626716597196500